app/controllers/comments_controller.rb

```diff
@@ -1,3 +1,18 @@
 class CommentsController < ApplicationController
-  def comments; end
+  def create
+    post = Post.find(params[:post_id])
+    comment = post.comments.new(text: comment_params[:text], user: current_user)
+
+    respond_to do |format|
```

Comment:
- [ ] The comments controller should have new actions similar to the one shown below. The `new` is used with the HTTP verb GET and creates an object instance.

```ruby

def new
  @comment = Comment.new
end
```

Reply:
Done ✅